src/main/java/core/basesyntax/RightTriangleSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RightTriangleSupplier {

```

Comment:
The class name 'RightTriangleSupplier' is not consistent with the task requirement, it should be 'FigureSupplier' as it should generate a variety of figures, not just right triangles.

Reply:
There is FigureSupplier. RightTriangleSupplier is used inside the FigureSupplier.